package.json

```diff
@@ -11,8 +12,16 @@
     "gulp-livereload": "~2.1.0",
     "gulp-plumber": "~0.6.3",
     "gulp-serve": "~0.2.0",
+    "insert-css": "^0.2.0",
     "jquery": "^2.1.3",
     "rimraf": "~2.2.8",
     "vinyl-source-stream": "^1.0.0"
+  },
+  "main": "src/index.coffee",
+  "browserify": {
+    "transform": ["coffeeify", "brfs"]
+  },
+  "scripts": {
+    "preview" : "browserify src/index.coffee -o ./preview/boxbits.js"
```

Comment:
I like the idea of skipping gulp for the `dist` generation. There isn't much the the current dist build anyway, so it makes sense to me. The preview workflow, though, is still pretty conveniently served by something like a gulpfile. It might be just scoped to that or something, I dunno. It's a separate piece of demo code anyway, and I was planning to move it out eventually.